slip-0044.md

```diff
@@ -1351,6 +1351,9 @@ All these constants are used as hardened derivation.
 | 20240430   | 0x8134d82e                    | NLK     | NuLinkCoin                        |
 | 608589380  | 0xa4465644                    | FVDC    | ForumCoin                         |
 | 1179993420 | 0xc655454c                    |         | Fuel                              |
+| 1179993431 | 0x464642F7                    | MTGBP   | MTGBP                             |
+| 1179993441 | 0x46464301                    | QFS     | Qfs                               |
+| 1179993421 | 0x464642ED                    | TTNC    | TakeTitan                         |
```

Comment:
1. This is not sorted according to value
2. Do not user upper-case letters in hex values
3. Hex values do not match the decimal values

Reply:
@prusnak  Please Check now I have made the changes according to your comments .

Reply:
No, the point 3 is not addressed

1179993421 is 0x4655454d not 0x464642ed for example

Reply:
@prusnak Thanks for updating me on the error.Now please check ..!!!

Reply:
the hex numbers are still wrong and do not match the numbers in the first column